include/usgscsm/UsgsAstroLsSensorModel.h

```diff
@@ -916,7 +916,38 @@ class UsgsAstroLsSensorModel : public csm::RasterGM, virtual public csm::Settabl
       double* achievedPrecision = NULL,
       csm::WarningList* warnings = NULL) const;
 
-   // This method computes the imaging locus.
+   // methods pulled out of los2ecf
+
+   void computeDistortedFocalPlaneCoordinates(
+       const double& lineUSGS, 
+       const double& sampleUSGS, 
+       double &distortedLine, 
+       double& distortedSample) const;

```

Comment:
We should get rid of all the USGS/isis variable names. I'd much rather see lineCCD, sampleCCD than lineUSGS, sampleUSGS.